wlauto/common/android/device.py

```diff
@@ -670,6 +670,19 @@ def get_device_model(self):
         except KeyError:
             return None
 
+    def is_network_connected(self):
+            network_host = 'www.google.com'
```

Comment:
`network_host` should probably be a parameter defaulting to `'www.google.com'` (also -- is there not a dedicated test domain for these things)?